text/0005-architecture-reviews.md

```diff
@@ -0,0 +1,135 @@
+# Architecture Review Process
+
+## Introduction
+
+Architectural Review decisions within the Firefox organization fall into two categories: "Roadmap" and “Design”.
+
+Product Reviews are not the scope of this process. This process is about architectural change, which should ultimately be in support of product features, but will generally have an effect on multiple product features. Architectural change can be growing new skills or in reducing large scale technical debt.
+
+The function of a **Roadmap Review** is to decide if a thing should be done. The goal is to bring together a packet of data to inform a management decision to provide resources to make the thing happen. A Roadmap Review should happen early in the process so that build time isn’t wasted on a "No" decision, but so that enough information is available to management.
```

Comment:
I think the Etsy process is much more closely aligned to a Design Review. "The ARWG’s role was _not_ to make a decision" (emphasis theirs), which is the opposite of a Roadmap review.